### PR TITLE
Ascent nerf & fix

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About the Pull Request

- Ascent lance had armor penetration nerfed from 50 to 40.
- Ascent sleeper icon is fixed.
- Ascent sleeper chems are in base chems list, instead of being put in Initialize(). Dexalin is removed due to being useless for Ascent.

## Why It's Good For The Game

- No more 111 damage per burst ignoring all armor.
- A fix.
- Useless initialize replaced. Dexalin is useless to Ascent, I think.

## Did you test it?

A bit, yes.

## Authorship

Me.

## Changelog

:cl:
tweak: Ascent lance had its armor penetration slightly reduced.
tweak: Dexalin removed from the pool of chemicals available in Ascent sleepers.
fix: Ascent sleeper icons have been fixed.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/projectile/beams.dm [hunk 1]
```diff
@@ -263,12 +263,12 @@
...
 	icon_state = "particle"
-	damage = 35
-	armor_penetration = 50
+	damage = 25
+	armor_penetration = 35
 	muzzle_type = /obj/effect/projectile/muzzle/particle
 	tracer_type = /obj/effect/projectile/tracer/particle
 	impact_type = /obj/effect/projectile/impact/particle
-	penetration_modifier = 0.5
+	penetration_modifier = 0.4
```
Comment (Cupax3) [inline code/modules/projectiles/projectile/beams.dm]: Please reverse the nerf. The particle lance is a Gyne (so whitelisted player) weapon, who should be aware when its usage is applicable. If they are abusing it, report them to yours truly

File: code/modules/projectiles/guns/energy/mantid.dm [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	icon_state = "particle_rifle"
 	item_state = "particle_rifle"
 	slot_flags = SLOT_BACK
-	force = 25 // Heavy as Hell.
+	force = 20 // Heavy as Hell.
```
Comment (Cupax3) [inline code/modules/projectiles/guns/energy/mantid.dm]: See other comment

File: code/modules/ascent/ascent_machines.dm [hunk 3]
```diff
@@ -72,11 +72,7 @@ MANTIDIFY(/obj/machinery/door/airlock/external/bolted, "mantid airlock", "door")
 	icon = 'icons/obj/ascent_sleepers.dmi'
 	base_type = /obj/machinery/sleeper
 	construct_state = /decl/machine_construction/default/no_deconstruct
-
-/obj/machinery/sleeper/ascent/Initialize(mapload, d, populate_parts)
-	. = ..()
-	base_chemicals["Crystalizing Agent"] = /datum/reagent/crystallizing_agent
-	base_chemicals["Bromide"] = /datum/reagent/toxin/bromide
+	base_chemicals = list("Inaprovaline" = /datum/reagent/medicine/inaprovaline, "Paracetamol" = /datum/reagent/medicine/painkiller/paracetamol, "Dylovene" = /datum/reagent/medicine/dylovene, "Bromide" = /datum/reagent/toxin/bromide, "Crystalizing Agent" = /datum/reagent/crystallizing_agent)
```
Comment (Cupax3) [inline code/modules/ascent/ascent_machines.dm]: More of a nitpick but imo the line-by-line is more readable, could you change your fix to that?

**Review by Cupax3 — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/projectile/beams.dm (hunk 1 above)
Comment (EgorDinamit) [reply code/modules/projectiles/projectile/beams.dm]: I just feel like 120 damage per 1-2 seconds is a bit too much.

Comment (Cupax3) [reply code/modules/projectiles/projectile/beams.dm]: A Gyne should rarely, if at all, be put into a position to fight or use the lance. That's what disposable alates are for.

**Comment by EgorDinamit** [comment]
Updated.

File: code/modules/ascent/ascent_machines.dm [hunk 4]
```diff
@@ -72,11 +72,13 @@ MANTIDIFY(/obj/machinery/door/airlock/external/bolted, "mantid airlock", "door")
...
-	. = ..()
-	base_chemicals["Crystalizing Agent"] = /datum/reagent/crystallizing_agent
-	base_chemicals["Bromide"] = /datum/reagent/toxin/bromide
+	base_chemicals = list(
+					"Inaprovaline" = /datum/reagent/medicine/inaprovaline,
+					"Paracetamol" = /datum/reagent/medicine/painkiller/paracetamol,
+					"Dylovene" = /datum/reagent/medicine/dylovene,
+					"Bromide" = /datum/reagent/toxin/bromide,
+					"Crystalizing Agent" = /datum/reagent/crystallizing_agent
+					)
```
Comment (Cupax3) [inline code/modules/ascent/ascent_machines.dm:81]: based

**Review by Cupax3 — APPROVED** [review]
approved provided CI clears

**Pull request merged**